bedrock/base/templates/macros.html

```diff
@@ -377,6 +377,16 @@
 </header>{#-- /#fxosnav-header --#}
 {%- endmacro %}
 
+{% macro google_play_button(alt_href='', class_name='', data_alt='', extra_attributes=None, id='', target_blank=False) -%}
```

Comment:
While this works, `data_alt` is probably something that's a one off name, chosen randomly for a specific page. Instead of explicitly defining it here, I think it may be more robust to allow passing optional attributes on the link element (similar to what we floated previously as an idea). I'm happy to use a different parameter for these link attributes so as to not mix them up with the ones used for `img`. We could handle `target_blank` here in the same way then too, instead of needing to define it here.